alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts

```diff
@@ -98,7 +100,7 @@ export class BoardManagementDialogComponent implements OnInit {
     this.step++;
     //We don't support creating Notifications
```

Comment:
We should probably update this comment too

Reply:
Comment removed.